ee/tabby-db/migrations/07-email-service-credentials/down.sql

```diff
@@ -1 +1 @@
-DROP TABLE email_service_credential;
+DROP TABLE email_settings;
```

Comment:
```suggestion
DROP TABLE email_setting;
```

Since we'll only have one row of email setting